{"message":"Error extracting metadata for D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.Abstractions/Nordic.Abstractions.csproj,D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.Cli/Nordic.Cli.csproj,D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.MSTest/Nordic.MSTest.csproj,D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.Runtime/Nordic.Runtime.csproj,D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.Simulation.AdaptedFriis/Nordic.Simulation.AdaptedFriis.csproj,D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.Simulation.Battery/Nordic.Simulation.Battery.csproj,D:/Adrian_w7nb0422/_Projekte/github/_NordicSim/NordicSim/src/Nordic.Simulation.MeshNetwork/Nordic.Simulation.MeshNetwork.csproj: System.IO.IOException: Der Prozess kann nicht auf die Datei \"D:\\Adrian_w7nb0422\\_Projekte\\github\\_NordicSim\\NordicSim\\src\\Nordic.Docu\\api\\Nordic.Abstractions.yml\" zugreifen, da sie von einem anderen Prozess verwendet wird.\r\n   bei System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)\r\n   bei System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)\r\n   bei System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize)\r\n   bei Microsoft.DocAsCode.Plugins.RootedFileAbstractLayer.Create(String file)\r\n   bei Microsoft.DocAsCode.Common.YamlUtility.Serialize(String path, Object graph, String comments)\r\n   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<ResolveAndExportYamlMetadata>d__19.MoveNext()\r\n   bei System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)\r\n   bei System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)\r\n   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<SaveAllMembersFromCacheAsync>d__13.MoveNext()\r\n--- Ende der Stapelüberwachung vom vorhergehenden Ort, an dem die Ausnahme ausgelöst wurde ---\r\n   bei System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   bei System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   bei Microsoft.DocAsCode.Metadata.ManagedReference.ExtractMetadataWorker.<ExtractMetadataAsync>d__11.MoveNext()","date_time":"2021-04-29T18:24:48.824821Z","message_severity":"error","code":"FatalError","correlation_id":"E1C27D43-DFC2-45A1-8F5B-49F4DE8F11BE.2"}
